seatunnel-api/src/main/java/org/apache/seatunnel/api/serialization/DeserializationSchema.java

```diff
@@ -35,9 +35,13 @@
     T deserialize(byte[] message) throws IOException;
 
     default void deserialize(byte[] message, Collector<T> out) throws IOException {
-        T deserialize = deserialize(message);
-        if (deserialize != null) {
-            out.collect(deserialize);
+        try {
+            T deserialize = deserialize(message);
+            if (deserialize != null) {
+                out.collect(deserialize);
+            }
+        } catch (Exception e) {
```

Comment:
print the dirty data?

Reply:
Dirty data has been printed through the warn log.

<img width="827" alt="image" src="https://user-images.githubusercontent.com/21999616/226159570-5b353331-bcbf-4d57-821c-a4758c3714b2.png">